pymc/tests/test_sampling.py

```diff
@@ -1606,7 +1606,7 @@ def test_linear_model(self):
             beta = pm.Normal("beta", 0, 0.1)
             mu = pm.Deterministic("mu", alpha + beta * x)
             sigma = pm.HalfNormal("sigma", 0.1)
-            obs = pm.Normal("obs", mu, sigma, observed=y)
+            obs = pm.Normal("obs", mu, sigma, observed=y, shape=x.shape)
```

Comment:
It's just so that we don't get an extra graph dependency and not have to change the tests expectations. I could equally have kept the original graph and add the new expected variable in the checks below. This was faster.